be/test/storage/rowset/rowset_test.cpp

```diff
@@ -607,8 +607,12 @@ TEST_F(RowsetTest, FinalMergeVerticalPartialTest) {
 
     ASSERT_TRUE(tablet->rowset_commit(2, rowset).ok());
     EXPECT_EQ(rows_per_segment * 2, read_tablet_and_compare(tablet, partial_schema, 2, rows_per_segment * 2));
+<<<<<<< HEAD
```

Comment:
remove it

Reply:
sorry, forgot save all at last. changed.